tests/schemabuilder/builder_test.py

```diff
@@ -104,6 +101,15 @@ def test_servers(self):
         assert 1 == len(servers)
         assert 'http://api.github.com' == servers[0]['url']
 
+    def test_pokeapi_schema_valid(self):
+        # this should conflate to
+        # /pokemon
+        # /abilities/*
+        # /types/*
+        # /pokemon/*
+        # meaning that it should recognize wildcards
+        # from all these paths
+        assert 4 == len(self.pokeapi_schema['paths'])
```

Comment:
It might be a bit clearer if this was an explicit assert that the paths are as expected? Or is the problem that there are random identifiers like `{fasd}` in the paths?

Reply:
There are random identifiers, but we could explicitly assert that the random identifiers are there, I'll change that.